<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yeelink API测试</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>U-ApiKey</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_text_uapikey">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>网址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_text_file_path">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_button_select_file">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选取</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>641</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>数据</string>
   </property>
   <widget class="QPlainTextEdit" name="ui_text_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ui_button_send_file">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>150</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件(POST)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_button_send_text">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文本(POST)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_button_send_text_hex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>250</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文本hex(POST)</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_button_send">
   <property name="geometry">
    <rect>
     <x>662</x>
     <y>300</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>仅发送(GET)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>641</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>调试信息</string>
   </property>
   <widget class="QPlainTextEdit" name="ui_text_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>621</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="ui_text_url">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_button_send_delete">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>仅发送(DELETE)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ui_button_send_text_put">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文本(PUT)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
